### PR TITLE
[RHELC-1615] CentOS Stream 9 test coverage

### PR DESCRIPTION
**Pull request by danmyway**
* Include CentOS Stream 9 in the integration tests coverage

Jira Issues:

- [RHELC-1615](https://issues.redhat.com/browse/RHELC-1615)

Checklist

- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [ ] `[RHELC-]` or `[HMS-]` is part of the PR title 
- [ ] GitHub label has been added to help with Release notes 
- [ ] PR title explains the change from the user's point of view
- [ ] Code and tests are documented properly
- [ ] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

File: tests/integration/common/checks-after-conversion/main.fmf [hunk 1]
```diff
@@ -34,6 +34,10 @@ order: 52
         test: pytest -m test_correct_distro
 
     /grub_default:
+        adjust+:
+          enabled: false
+          when: distro == stream-9-latest
+          because: The versions of latest RHEL kernel < latest Stream kernel, which is not handled very well in the code.
```
Comment (kokesak) [inline tests/integration/common/checks-after-conversion/main.fmf:40]: Do we have some jira ticket to link here?

Comment (danmyway) [reply tests/integration/common/checks-after-conversion/main.fmf:40]: It's this one https://issues.redhat.com/browse/RHELC-1717
I'll attach it in the comment.

Comment (kokesak) [reply tests/integration/common/checks-after-conversion/main.fmf]: Yes please attach that there. Maybe even use the keyword `link: https://issues.redhat.com/browse/RHELC-1717`

**Review by kokesak — APPROVED** [review]

**Review by hosekadam — APPROVED** [review]
Looks good

**Pull request merged**